daffodil-test/src/test/resources/org/apache/daffodil/section12/length_properties/LengthProperties.tdml

```diff
@@ -911,7 +911,7 @@
     
     <xs:element name="bit26" type="xs:short" dfdl:lengthKind="explicit" dfdl:length="5" dfdl:byteOrder="bigEndian" />
     <xs:element name="bit27" type="xs:int" dfdl:lengthKind="explicit" dfdl:length="9" dfdl:byteOrder="bigEndian" />
-    <xs:element name="bit28" type="xs:integer" dfdl:lengthKind="explicit" dfdl:length="13" dfdl:byteOrder="bigEndian" />
+    <xs:element name="bit28" type="xs:int" dfdl:lengthKind="explicit" dfdl:length="13" dfdl:byteOrder="bigEndian" />
```

Comment:
Lots of schemas from 3rd parties may run in to this issue. I have seen schemas where the author tends to use the more general type xs:integer thinking it is equivalent to all the smaller types. 

I think we need to look beyond just Daffodil's tests to see the impact of this change.